content/posts/consumindo_web_service_no_android.md

```diff
@@ -0,0 +1,462 @@
+
+Para exemplo do post, vamos consumir um serviço para buscas de CEP, os passos que deveriamos seguir são:
+
+- Desenhar o *layout*
+- Consumir nosso serviço
+- Mostrar os dados consumidos para o usuário
+
+Após concluir todos esses passos teremos nossa app funcionando da seguinte maneira:
+
+![Exemplo consumir serviço de CEP](http://blog.matheuscastiglioni.com.br/arquivo/download/arquivos-imagens/2017/11/buscando-cep-requisicao-http.gif)
+
+## Desenhando nossa app
+
+O primeiro passo será criar o *layout* da nossa app, como o assunto do *post* refere-se ao *web service*, irei disponibilizar o layout:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:padding="50dp"
+    android:layout_width="match_parent"
+    tools:context="br.com.matheuscastiglioni.blog.requisicao_http.MainActivity">
+
+    <EditText
+        android:digits="0123456789"
+        android:layout_height="wrap_content"
+        android:hint="CEP"
+        android:id="@+id/etMain_cep"
+        android:textColor="#595959"
+        android:textSize="25sp"
+        android:inputType="number"
+        android:layout_width="match_parent"/>
+
+    <LinearLayout
+        android:gravity="center"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:layout_width="match_parent">
+
+        <Button
+            android:background="@color/colorPrimary"
+            android:layout_height="wrap_content"
+            android:id="@+id/btnMain_buscarCep"
+            android:layout_marginBottom="10dp"
+            android:padding="10dp"
+            android:text="Buscar CEP"
+            android:textColor="#FDFDFD"
+            android:textSize="22sp"
+            android:layout_width="wrap_content"/>
+
+    </LinearLayout>
+
+    <TextView
+        android:layout_height="match_parent"
+        android:id="@+id/etMain_resposta"
+        android:textColor="#595959"
+        android:textSize="20sp"
+        android:layout_width="match_parent"/>
+
+</LinearLayout>
+```
+
+Após adicionar esse *layout* para a *activity*, devemos ter nossa app parecido com:
+
+![App para consumir CEP](http://blog.matheuscastiglioni.com.br/arquivo/download/arquivos-imagens/2017/11/buscando-cep-requisicao-http.png)
+
+## Consumindo um serviço
+
+Agora que o *layout* esta pronto, podemos começar a consumir nosso serviço, para isso alguns passos devem ser seguidos, sendo eles:
+
+- Adicionar um listener no botão **Buscar CEP**
```

Comment:
Se você está colocando as palavras em inglês em itálico, precisa aqui em *listener* também.